src/test/scala/BindingTests.scala

```diff
@@ -187,4 +187,65 @@ class BindingTests extends FunSuite {
   test("EntityRefMap arrays are equaly sized") {
     assert(EntityRefMap.keys.size == EntityRefMap.values.size)
   }
+
+  test("onClick = Function0") {
+    var clicked = false
+    val button  = <button onclick={ () => clicked = true }>Click Me!</button>
```

Comment:
Function0, nice 👍 Surprisingly often, you don't care about the type of the event.